web_client/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import MechanicalTheatre from './MechanicalTheatre';
+import Passage from './Passage';
+import './App.css';
+
+const FINNEGAN_FOREVER_URL = "https://drsvfxtdvi.execute-api.us-east-1.amazonaws.com/dev/passage";
+
+class App extends Component {
+    render() {
+        return (
+            <div className="App">
+              <MechanicalTheatre>
+                <Passage finneganForeverUrl={FINNEGAN_FOREVER_URL}/>
```

Comment:
A little confused re: our conversation earlier. Is the idea here to do:

```jsx
<IntervalTextReader readText=readText>
    <Passage /> // <--- the style component
</ IntervalTestReader>
```

@evanhammer

Reply:
yeah, that's one approach. You'll have to add props to `Passage` within IntervalTextReader. The other is a higher order component: `const IntervalTextReader = intervalTextReader(Passage);`

https://medium.com/ingenious/a-tale-of-higher-order-components-render-props-a1ba47e8cfeb

Reply:
Sweet that seems like it makes more sense